node/db/src/rocks.rs

```diff
@@ -151,3 +152,35 @@ impl Blockstore for RocksDb {
         self.bulk_write(&values).map_err(|e| e.into())
     }
 }
+
+impl BitswapStore for RocksDb {
+    /// fvm_ipld_encoding::DAG_CBOR(0x71) is covered by [libipld::DefaultParams]
```

Comment:
Mark code blocks to let the spellchecker know that these aren't just weirdly spelled words.

```suggestion
    /// `fvm_ipld_encoding::DAG_CBOR(0x71)` is covered by [`libipld::DefaultParams`]
```

Reply:
fixed